Makefile

```diff
@@ -0,0 +1,82 @@
+SHELL ?= /bin/bash
+
+.DEFAULT_GOAL := build
+
+################################################################################
+# Version details                                                              #
+################################################################################
+
+# This will reliably return the short SHA1 of HEAD or, if the working directory
+# is dirty, will return that + "-dirty"
+GIT_VERSION = $(shell git describe --always --abbrev=7 --dirty --match=NeVeRmAtCh)
+
+################################################################################
+# Containerized development environment-- or lack thereof                      #
+################################################################################
+
+ifneq ($(SKIP_DOCKER),true)
+	PROJECT_ROOT := $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+	# https://github.com/krancour/go-tools
+	# https://hub.docker.com/repository/docker/krancour/go-tools
+	GO_DEV_IMAGE := krancour/go-tools:v0.3.0
```

Comment:
Definitely not a pressing matter, but it would be great if we had an org-level project that had all the tools we needed.

Reply:
Agreed 100%. I will commit to having that done before 2.0 is GA. I just hadn't wanted to start there.